src/test/regress/expected/multi_insert_select.out

```diff
@@ -1477,9 +1475,7 @@ SELECT user_id FROM raw_events_first WHERE user_id = 101;
 (1 row)
 
 ROLLBACK;
--- Copy after insert is disallowed since the insert is not immediately
--- committed and the copy uses different connections that will not yet
--- see the result of the insert.
+-- Copy after insert can be used in the same transaction.
```

Comment:
This still shouldn't work since it's a multi-shard INSERT.. SELECT, only single-shard INSERT..SELECT is compatible.